### ISSUE TITLE
USWDS - File input: Remove aria-disabled from from parent wrapper

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5144]
Currently, the file input component adds `aria-disabled` to its parent wrapper, `usa-file-input`. In JAWS, this prevents the label from being read, which violates best practice. We should remove the `aria-disabled` attribute from `usa-file-input` element and confirm that the removal does not cause harm.

**Comment by thisisdano** [comment]
Fixed by https://github.com/uswds/uswds/pull/5063
